docs/measurements/physics/calibration.md

```diff
@@ -0,0 +1,75 @@
+
+# BPM Calibration
+
+Beam Position Monitors (BPM) are instruments used to measure the transverse position of the beam at a certain longitudinal position in the accelerator.
+However, the returned values are often off by a certain factor called calibration factor.
```

Comment:
maybe adding a simple formula here and/or a link to Dr. Carlier Msc Thesis, where there is a section on that https://cds.cern.ch/record/2057179/files/CERN-THESIS-2015-154.pdf?

Reply:
-> make an issue